verticapy/performance/vertica/qprof_interface.py

```diff
@@ -137,13 +137,22 @@ def get_qplan_tree(self, use_javascript=True):
         """
         self.use_javascript = use_javascript
         # widget for choosing the metrics
-        tags = widgets.TagsInput(
-            value=["Execution time in ms", "Produced row count"],
-            allowed_tags=[
-                QprofUtility._get_metrics_name(i) for i in QprofUtility._get_metrics()
-            ],
-            allow_duplicates=False,
+        options_dropwdown = [
+            QprofUtility._get_metrics_name(i) for i in QprofUtility._get_metrics()
+        ]
+        dropdown1 = widgets.Dropdown(
```

Comment:
A few questions about the dropdown choice:
 - Why do you think it is better than a single `TagsInput`?
 - Any plan of selecting more than 2 metrics at a time? If yes then dropdowns will not scale well.
 - If we end up using dropdowns for metrics, we should prevent setting the same metric in both dropdowns

Reply:
Because of user feedback. They thought it was not obvious/intuitive how to change the values. And the dropdown is more intuitive. 
For now, we do not plan to do that. If we did, it would be straightforward to add another dropdown. 
I think we should not restrict user from same selection because it could create a bad user experience e.g.

User wants to see row count (dropdown # 1) and time in ms (dropdown # 2). But then they decide to swap it. Now if they want to make dropdown # 1 = time in ms, it will not allow. That is an interim step they need to perform in order to switch it.

Reply:
> User wants to see row count (dropdown # 1) and time in ms (dropdown # 2). But then they decide to swap it. Now if they want to make dropdown # 1 = time in ms, it will not allow. That is an interim step they need to perform in order to switch it.
Got you. But they should also not be able to set the same value on both dropdowns. ex: dropdown # 1 = time in ms and dropdown # 2 = time in ms. How are you going to handle this case?